### ISSUE TITLE
Какие ещё методы нужны для справочника?

### ISSUE THREAD

**Issue by sanmai** [sanmai/cdek-info#3]
Сейчас у справочников есть только методы для перечисления составляющих. Какие ещё методы нужны?

Нужен ли метод для получения описания по ID?

Похожая функция нужна была @iamwildtuna.

**Comment by iamwildtuna** [comment]
Да, логично сделать метод, который будет возвращать текстовое представление статуса по его ID, чтобы не хранить кучу текста у себя в базе
